portfolio/src/test/java/com/google/sps/CampaignServletTest.java

```diff
@@ -102,7 +116,7 @@ public void getValueFromJSONTest3() {
 	}
 
 	@Test
-	public void processJSONTest1() {
+	public void processJSON1_simple() {
 		MockCampaignsServlet testServlet = new MockCampaignsServlet();
 		String inputString = getStringFromFile("input/processJSON1.txt");
 		String result = testServlet.processJSON(inputString);
```

Comment:
test missing assert. same through every test that doesn't have an assert. if it expects an error, assert that we should receive an error